controller/newspaper.js

```diff
@@ -24,7 +24,7 @@ exports.patchArticleById = (req, res, next) => {
     updateArticleById(newVotes, articleId).then((article)=> {
         res.status(200).send({updatedArticle: article});
```

Comment:
quick note on line 25 - I'd try and keep the keys we send data back on uniform - so it should ideally be `article` instead of `updatedArticle` 

This is mainly opinion-based and I'm happy for you to take your own path, but from an API perspective you'd prefer to keep it all uniform and standard